Bottom: 9ccf33a67f2d07fca0bbb5c2e794ac7a95efe147
Top:    da6197b9698c544733a22189c2c2dda054a284a8
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 10:19:05 -0500

Build ntel whenever any layer changes

Works by adding an extra step to ntel builds that checks the
change sourestamps and checks out changed layers. Potentially this
could detect changes to multiple layers if they fall within the
tree stable timeout.

This really only works for layers in the same group on gitlab, so any
repository under /ntel/.


---
diff --git a/roles/buildbot-master/files/master.cfg b/roles/buildbot-master/files/master.cfg
index 1245dc16..e1608831 100644
--- a/roles/buildbot-master/files/master.cfg
+++ b/roles/buildbot-master/files/master.cfg
@@ -170,7 +170,7 @@ def codebaseGenerator(chdict):
     """
 
     giturl = giturlparse(chdict['repository'])
-    return "%s/%s" % (giturl.owner, giturl.repo)
+    return giturl.owner
 
     # path = urlparse(chdict['revlink']).path.lstrip('/')
 
diff --git a/roles/buildbot-master/files/openembedded.py b/roles/buildbot-master/files/openembedded.py
index 5cbd252a..9166ece0 100755
--- a/roles/buildbot-master/files/openembedded.py
+++ b/roles/buildbot-master/files/openembedded.py
@@ -9,7 +9,7 @@ c = WorkerConfig = {}
 
 
 DEFAULT_BBFLAGS = '-k'
-DEFAULT_CODEBASE = "ntel/setup-scripts"
+DEFAULT_CODEBASE = "ntel"
 
 # Workers
 # The 'workers' list defines the set of recognized buildworkers. Each element is
